bin/compress_posts

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bin/rails runner
+
+count = Post.where("html_body IS NOT NULL AND text_body IS NOT NULL").count
+$stdout.puts "Found #{count} posts to compress"
+counter = 0
+
+Post.where("html_body IS NOT NULL AND text_body IS NOT NULL").find_each do |post|
+  if post.id > counter
```

Comment:
uhhh what is `counter` doing for us here?

Reply:
I originally wrote it to only print every 100 posts, but then it was so slow I gave up 🫠